qiita_ware/ebi.py

```diff
@@ -674,25 +644,7 @@ def send_xml(self):
                 study_accession = accessions.group('study')
                 submission_accession = accessions.group('submission')
 
-                LogEntry.create('Runtime', "Study accession:\t%s" %
-                                study_accession)
-                LogEntry.create('Runtime', "Submission accession:\t%s" %
-                                submission_accession)
-
-                print "Study accession:\t", study_accession
-                print "Submission accession:\t", submission_accession
-            else:
-                LogEntry.create('Runtime', ("However, the accession numbers "
-                                            "could not be found in the output "
-                                            "above."))
-                print ("However, the accession numbers could not be found in "
-                       "the output above.")
-        else:
-            LogEntry.create('Fatal', curl_result)
-            print curl_result
-            print "FAILED"
-
-        return (study_accession, submission_accession)
+        return study_accession, submission_accession
```

Comment:
Are you sure though? It complicates the method to be both a parser and to care about errors. I'd vote to keep the parser as simple as necessary. For instance, `parse_fasta` doesn't care if the sequences have random characters as the validation and handling of the parsed records is at the responsibility of the consumer (or another method call issued by the consumer).